src/components/OfflineSearch.ts

```diff
@@ -47,9 +46,10 @@ export default class OfflineSearch extends Component<OfflineSearchProps, Offline
     private renderBar(): ReactElement<SearchBarProps> {
         if (this.state.validationPassed) {
             return createElement(SearchBar, {
-                ...this.props as CommonProps,
+                defaultQuery: this.props.defaultQuery,
                 onTextChangeAction: this.updateConstraints,
-                style: parseStyle(this.props.style)
```

Comment:
-Since you have removed Common style and css addition from here. please make sure it is supported in the widget. Perhaps to move it to the container

Reply:
done